From: Niko Tyni <ntyni@debian.org>
Date: Wed, 1 Jul 2015 22:23:53 +0300
Subject: Make EU::MakeMaker honour MANnEXT settings in generated manpage
 headers

This was inherited from early changes by Brendan O'Dea, previously
(accidentally) in debian/prefix_changes.diff and before that in
debian/extutils_hacks.diff

Bug-Debian: https://bugs.debian.org/247370
---
 cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm | 2 +-
 cpan/ExtUtils-MakeMaker/t/build_man.t          | 8 ++++----
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm b/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm
index 826a4a5..92f841f 100644
--- a/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm
+++ b/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm
@@ -1091,7 +1091,7 @@ END
     my @man_cmds;
     foreach my $num (qw(1 3)) {
         my $pods = $self->{"MAN${num}PODS"};
-        my $p2m = sprintf <<'CMD', "\$(MAN${num}SECTION)", "$]" > 5.008 ? " -u" : "";
+        my $p2m = sprintf <<'CMD', "\$(MAN${num}EXT)", "$]" > 5.008 ? " -u" : "";
 	$(NOECHO) $(POD2MAN) --section=%s --perm_rw=$(PERM_RW)%s
 CMD
         push @man_cmds, $self->split_command($p2m, map {($_,$pods->{$_})} sort keys %$pods);
diff --git a/cpan/ExtUtils-MakeMaker/t/build_man.t b/cpan/ExtUtils-MakeMaker/t/build_man.t
index 59a80eb..d525e2a 100644
--- a/cpan/ExtUtils-MakeMaker/t/build_man.t
+++ b/cpan/ExtUtils-MakeMaker/t/build_man.t
@@ -222,9 +222,9 @@ unlink $README;
         like $makefile, qr/\QMAN1SECTION = 1pm\E/xms, "Set MAN1SECTION";
         like $makefile, qr/\QMAN3SECTION = 3pm\E/xms, "Set MAN3SECTION";
 
-        like $makefile, qr/\Q$(POD2MAN) --section=$(MAN1SECTION) \E/,
-            "Set POD2MAN section to \$(MAN1SECTION)";
-        like $makefile, qr/\Q$(POD2MAN) --section=$(MAN3SECTION) \E/,
-            "Set POD2MAN section to \$(MAN3SECTION)";
+        like $makefile, qr/\Q$(POD2MAN) --section=$(MAN1EXT) \E/,
+            "Set POD2MAN section to \$(MAN1EXT)";
+        like $makefile, qr/\Q$(POD2MAN) --section=$(MAN3EXT) \E/,
+            "Set POD2MAN section to \$(MAN3EXT)";
     }
 }
